docs/mpid_messaging.md

```diff
@@ -46,7 +48,7 @@ The network inbox is an even simpler structure and will be again named with the
 
 ```c++
 struct MpidAlert {
-  Identity message_id,
+  Identity message_id;
   PublicMpid::Name sender;
```

Comment:
it seems according the description of messaging step 2 (sending MpidHeader),
message_head shall be part of the member data of MpidAlert struct